gcloud/pubsub/subscription.py

```diff
@@ -305,8 +305,10 @@ def set_iam_policy(self, policy, client=None):
         client = self._require_client(client)
         path = '%s:setIamPolicy' % (self.path,)
         resource = policy.to_api_repr()
+        # 'set_iam_policy' API requires an extra wrapper. :(
```

Comment:
The extra element is utterly pointless, counter-intuitive and un-RESTful:  other than that, I have no complaints. :)